src/pages/index.astro

```diff
@@ -20,6 +20,6 @@ import Post from "../components/Post.vue";
     <title>Canopas Blogs</title>
   </head>
   <body class="select-none">
-    <Post />
+    <Post client:visible />
```

Comment:
What is use of it?

Reply:
component javascript code is never accessible and visible. so, that's why i added